### PR TITLE
Issue 82

### PR DESCRIPTION
**Pull request by cavukovic**
### Description
Button for deleting a user is no longer there for the logged-in user

### Screenshots / Demo Video
Screen Shot 2022-06-28 at 12 49 12 PM

### Testing
Go to users tab and see if the delete button is there for the user that you are logged in under 

### Checklist
- [x] Provided ticket and description
- [x] Your PR title contains the ISSUE ticket number e.g "86: ..."
- [x] Provided screenshots/demo
- [x] Provided testing information
- [x] Provided adequate test coverage for all new code
- [x] Added PR reviewers
- [x] Ensure at least 1 review before merging

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (User1m) [inline .vscode/launch.json:1]: lets not commit this file `git rm --cached .editorconfig`

File: packages/client/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (User1m) [inline packages/client/package-lock.json:1]: lets not commit this file `git rm --cached packages/client/package-lock.json`

File: packages/server/seeds/dev/index.js [hunk 3]
```diff
@@ -12,8 +12,9 @@ const tenants = require('./ref/tenants');
 const usdrAgency = agencies.find((a) => a.abbreviation === 'USDR');
 // const nevadaAgency = agencies.find((a) => a.abbreviation === 'NV');
 
+console.log(process.env.NODEMAILER_EMAIL_PW);
```
Comment (User1m) [inline packages/server/seeds/dev/index.js:15]: nit: rm console logs

**Comment by User1m** [comment]
@cavukovic Comments needing addressed

File: packages/server/seeds/dev/index.js (hunk 3 above)
Comment (cavukovic) [reply packages/server/seeds/dev/index.js:15]: fixed

File: .vscode/launch.json (hunk 1 above)
Comment (cavukovic) [reply .vscode/launch.json:1]: removed .editorconfig

Comment (User1m) [reply .vscode/launch.json:1]: sorry wrong file @cavukovic  `git rm --cached .vscode/launch.json`

**Pull request merged**